tests/e2e/framework/kubernetes.go

```diff
@@ -87,6 +87,8 @@ var (
 	galleyTag          = flag.String("galley_tag", os.Getenv("TAG"), "Galley tag")
 	sidecarInjectorHub = flag.String("sidecar_injector_hub", os.Getenv("HUB"), "Sidecar injector hub")
 	sidecarInjectorTag = flag.String("sidecar_injector_tag", os.Getenv("TAG"), "Sidecar injector tag")
+	cniHub             = flag.String("cni_hub", os.Getenv("HUB"), "CNI hub")
```

Comment:
remove this and line below. Let the helm chart default it.

Reply:
I removed the default value (actually set it to ""), but I didn't remove the line.